### PR TITLE
A note on fees blogpost 

### PR DESCRIPTION
**Pull request by dEBRUYNE-1**

### CODE REVIEW

File: _posts/2017-12-11-A-note-on-fees.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: A note on fees
+tags: [crypto, mrl, dev]
```
Comment (MaxXor) [inline _posts/2017-12-11-A-note-on-fees.md]: Please change tags to: `[crypto, research]`
There exist no `mrl` or `dev` tags.

Comment (dEBRUYNE-1) [reply _posts/2017-12-11-A-note-on-fees.md]: Fixed.

File: _posts/2017-12-11-A-note-on-fees.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+**NewReward = BaseReward - Penalty**  
+
+Where:  
+
+ -  M<sub>N</sub> is the median of the block size over the last N blocks, with N being [100](https://github.com/monero-project/bitmonero/blob/master/src/cryptonote_config.h#L57) in Monero  
+ -  BlockSize is the size of the current block  
+ -  BaseReward is the reward as per the emission curve or where applicable the tail emission  
+ -  NewReward is the actual reward paid to the miner  
+ -  The maximum allowed block size, BlockSize, is 2M<sub>N</sub>
```
Comment (stoffu) [inline _posts/2017-12-11-A-note-on-fees.md]: Isn't this bit `, BlockSize,` unnecessary?

Comment (dEBRUYNE-1) [reply _posts/2017-12-11-A-note-on-fees.md]: Why? Even though it's intuitive, I think it'd best to explain all variables of the formula.

Comment (stoffu) [reply _posts/2017-12-11-A-note-on-fees.md]: But the variable BlockSize is already explained above in the second bulletpoint item.

Comment (dEBRUYNE-1) [reply _posts/2017-12-11-A-note-on-fees.md]: I misinterpreted your earlier comment. It seems redundant indeed. Fixed & squashed.

File: _posts/2017-12-11-A-note-on-fees.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+Note that the formula of the BaseReward is defined [as follows](https://bitcointalk.org/index.php?topic=583449.0):  
+
+**BaseReward = 2 * ((S - A) * 2<sup>-20</sup> * 10<sup>-12</sup>)**  
+
+Where:  
+
+ -  2 is the adjustment factor for the switch to two minute blocks  
+ -  S is the initial number of atomic units is = 2<sup>64</sup> - 1  
+ -  A is the current circulation, which can be found [here](https://moneroblocks.info/)
```
Comment (rodolfo912) [inline _posts/2017-12-11-A-note-on-fees.md]: You could add here that the number of the current circulation (emission) on moneroblocks has to be multiplied with 10^12 in order to comply with S being in atomic units, or briefly explain atomic units in the definition of S. It took me (as a beginner) some time to figure out why my calculated BaseReward (with A = 15449232) was ~35 and not ~5.7XMR.

Comment (dEBRUYNE-1) [reply _posts/2017-12-11-A-note-on-fees.md]: Thanks for the suggestion. Added another line and squashed.

**Pull request merged**